tests/unit/Logger.js

```diff
@@ -256,3 +256,81 @@ describe('Werelogs Module-level Logger can log as specified by the log level', (
     it('Fatal level does not filter fatal level out',   filterGenerator('fatal', 'fatal'));
 });
 /* eslint-enable no-multi-spaces, max-len */
+
+describe('Werelogs Module-level Logger provide the DefaultFields logic', () => {
+    const loggerConfig = {
+        level: 'info',
+        dump: 'fatal',
+        end: 'info',
+    };
+
+    it('should not modify the object passed as a parameter', done => {
+        const defaultFields = {
+            name: 'TestDefaultFields1',
+            data: 0,
+        };
+        const add1 = {
+            attr1: 0,
+        };
+        const add2 = {
+            attr2: 'string',
+        };
+        const dummyLogger = new DummyLogger();
+        Config.simpleLogger = dummyLogger;
+        const logger = new Logger(defaultFields, loggerConfig);
+        logger.addDefaultFields(add1);
+        logger.addDefaultFields(add2);
+        assert.deepStrictEqual(add1, { attr1: 0 });
+        assert.deepStrictEqual(add2, { attr2: 'string' });
+        done();
+    });
+
+    it('should add one added default field to the log entries', done => {
+        const defaultFields = {
+            name: 'TestDefaultFields2',
+            data: 0,
+        };
+        const clientInfo = {
+            clientIP: '127.0.0.1',
+        };
+        const dummyLogger = new DummyLogger();
+        Config.simpleLogger = dummyLogger;
+        const logger = new Logger(defaultFields, loggerConfig);
+        logger.addDefaultFields(clientInfo);
+        logger.info('test message');
+        assert.strictEqual(clientInfo.clientIP,
```

Comment:
Indeed.
 - [ ] Check the order of the assert parameters (expected on the right)